src/routes/search/containers/__tests__/SearchPage.spec.js

```diff
@@ -88,7 +88,7 @@ describe('SearchPage', () => {
     const searchInputProps = tree.find('SearchInput').props()
 
     // the root category should not be returned
-    expect(searchPage.findCategories()).toHaveLength(categories.toArray().length - 1)
+    expect(searchPage.findCategories()).toHaveLength(categories.toArray().length)
```

Comment:
Why is this not the case anymore? At least the comment should be removed probably